### PR TITLE
ci: missing exports for import-external-cluster.sh

### PR DESCRIPTION
**Pull request by Xenon-777**
Add funktion for the missing export environment variable that needed for the import-external-cluster.sh script

### CODE REVIEW

**Review by leseb — CHANGES_REQUESTED** [review]
Thanks for your contribution, please address the CI issues.

**Comment by Xenon-777** [comment]
If you say my what a CI is.
Sorry, i am not a Developer, i am a Administrator.
The Change that i have committed are coming direct out of a rook install try.
PS: 
A Hind. Don't name second Cluster in main Cluster "external". That slave Cluster. External Cluster is not in same Cluster of the main Cluster. That has cost me Weeks to see through that nomenclature fail.

**Comment by leseb** [comment]
> If you say my what a CI is.
> Sorry, i am not a Developer, i am a Administrator.
> The Change that i have committed are coming direct out of a rook install try.
> PS:
> A Hind. Don't name second Cluster in main Cluster "external". That slave Cluster. External Cluster is not in same Cluster of the main Cluster. That has cost me Weeks to see through that nomenclature fail.

That's ok, please see: 

- https://github.com/rook/rook/pull/7443/checks?check_run_id=2146709132
- https://github.com/rook/rook/pull/7443/checks?check_run_id=2146708728
-  https://github.com/rook/rook/pull/7443/checks?check_run_id=2146709001

The first two is about formatting the commit message correctly, also read https://rook.io/docs/rook/master/development-flow.html#commit-structure

Thanks!

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh [hunk 1]
```diff
@@ -57,15 +62,25 @@ function createCephCSIKeyringCephFSProvisioner() {
...
 createCheckerKey
 createCephCSIKeyringRBDNode
 createCephCSIKeyringRBDProvisioner
 createCephCSIKeyringCephFSNode
 createCephCSIKeyringCephFSProvisioner
+getMonIP
+
+echo "export NAMESPACE=rook-ceph"
+echo ""
+echo "!! IMPORTENT !! Check if all exports correct, particularly NAMESPCAE and ROOK_EXTERNAL_FSID. (For intern slave Cluster must be NAMESPACE=rook-ceph-extern and ROOK_EXTERNAL_FSID=rook-ceph)"
```
Comment (BlaineEXE) [inline cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh:84]:
```suggestion
echo "!! IMPORTANT !! Check if all exports are correct, particularly NAMESPACE and ROOK_EXTERNAL_FSID. For default recommendations, internal cluster must be NAMESPACE=rook-ceph-extern and ROOK_EXTERNAL_FSID=rook-ceph"
```

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # this script creates all the users/keys on the external cluster
 # those keys will be injected via the import-external-cluster.sh once this one is done running
-# so you can run import-external-cluster.sh right after this script
+# so you can run import-external-cluster.sh right after this script
```
Comment (BlaineEXE) [inline cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh:4]:
```suggestion
# so you can run import-external-cluster.sh right after this script
```

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh [hunk 3]
```diff
@@ -57,15 +62,25 @@ function createCephCSIKeyringCephFSProvisioner() {
...
 checkEnv
+getFSID
 createCheckerKey
 createCephCSIKeyringRBDNode
 createCephCSIKeyringRBDProvisioner
 createCephCSIKeyringCephFSNode
 createCephCSIKeyringCephFSProvisioner
+getMonIP
+
+echo "export NAMESPACE=rook-ceph"
```
Comment (BlaineEXE) [inline cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh:82]: Why is the script just `echo`ing these exports?

**Review by BlaineEXE — CHANGES_REQUESTED** [review]

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh (hunk 3 above)
Comment (Xenon-777) [reply cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh:82]: Ömmmm ... the create-external-cluster-resources.sh should put out the exports for the import-external-cluster.sh. And the import-external-cluster.sh need this 3 exports too. I have not touch the import-external-cluster.sh script.

Comment (Xenon-777) [reply cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh:82]: PS: i have change this anyway. My change make no export anymore. The Problem from export is any User (Gast, low User) that can execute printenv can read the Token. My change echo the Variable Par only. then i can make:
kubectl -n rook-ceph exec -t ceph-toolbox --  create-external-cluster-resources.sh >cluster-var.conf. then i have a file that can insert with:
. cluster-var.conf
in the import-external-cluster.sh. This file i can make root only and 600 to hide it for unauthorized User.

**Comment by leseb** [comment]
Closing due to inactivity and review were never addressed. Feel free to re-open, thanks.

**Pull request closed without merging**